lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -262,6 +262,7 @@ class LegacyExportCommand < HammerCLIKatello::SingleResourceCommand
       end
     end
 
+    # rubocop:disable Metrics/ClassLength
```

Comment:
This is a sign. If we just disable Rubocop every time we want to do something nasty, why use it at all? I feel the same about the other `rubocop:disable` places. We should break up this class.

Reply:
I will do that on the next bug since I have to add some more code to the class, this time I found out I didnt need to disable that since I was still working on the pr. Will mark this one as resolved.